src/App.js

```diff
@@ -50,6 +48,8 @@ function App() {
               <Route path="/merchant/register" element={<MerchantRegister />} />
               <Route path="/merchant/login" element={<MerchantLoginPage />} />
               <Route path="/*" element={<PageNotFound />} />
+              <Route path="/register" element={<CustomerRegister />} />
+              <Route path="/login" element={<CustomerLoginPage />} />
```

Comment:
both these routes should come above "/*" route as "/*" route will catch all requests all the routes below it will not trigger anytime